content/en/docs/user-documentation/how-tos/offline-containers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Offline/disconnected container images
+draft: false
+weight: 9
+tags:
+  - Container Storage
+  - Disconnected
+description: Offline containers are containers which are stored in the operating system image, and made available to `cri-o` via the `/etc/container/storage.conf` `additionalimagestores` list.
+---
+## What are offline container images
+
+Offline containers are containers which are stored in the operating system image,
```

Comment:
"system image" sounds more specific to rpm-ostree installs than standard installs.

Reply:
I was explicitly thinking of that, may be I should extend the explanation to make it clearer it works for both.

Reply:
+1